### PR TITLE
Update finder.xml

### PR DESCRIPTION
**Pull request by ghazal**
Pull Request for Issue #11653 .
### Summary of Changes

Remove a call to a removed folder in finder.xml

### CODE REVIEW

File: administrator/components/com_finder/finder.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	</files>
 	<media destination="com_finder" folder="media">
 		<folder>js</folder>
-		<folder>images</folder>
+
```
Comment (brianteeman) [inline administrator/components/com_finder/finder.xml]: Can you remove this blank line please

**Comment by ghazal** [comment]
Sorry.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 48f812cfffa5ce1f2e4813b43f872f8194651dba<br /><br />On review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11656">issues.joomla.org/joomla-cms/11656</a>.</sub>

**Comment by dgt41** [comment]
I have tested this item :white_check_mark: successfully on 48f812cfffa5ce1f2e4813b43f872f8194651dba<br /><br />On code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11656">issues.joomla.org/joomla-cms/11656</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11656">issues.joomla.org/joomla-cms/11656</a>.</sub>

**Pull request merged**